plugins.json

```diff
@@ -415,5 +415,13 @@
         "pip_url": "aiida-wannier90-workflows",
         "plugin_info": "https://raw.github.com/aiidateam/aiida-wannier90-workflows/master/setup.json",
         "code_home": "https://github.com/aiidateam/aiida-wannier90-workflows"
+    },
+    "QECpWorkChain": {
+        "name": "aiida-QECpWorkChain",
+        "entry_point_prefix": "QECpWorkChain",
```

Comment:
The convention for the `entry_point_prefix` is that the entry points of the package all start with that prefix. Currently this is not the case. Your `setup.json` reads:
```
    "entry_points": {
        "aiida.workflows": [
            "QECpWorkChain = aiida_QECpWorkChain.workflow:CpWorkChain"
        ],
        "aiida.calculations": [
            "DeepMdTrain = aiida_QECpWorkChain.deepmd:DeepMdTrainCalculation"
        ],
        "aiida.parsers": [
            "DeepMdTrainParser = aiida_QECpWorkChain.deepmd:DeepMdTrainParser"
        ]
    },
```
An example entry point name is `QECpWorkChain` in the group `aiida.workflows`. Note how it does not start with `QECpWorkChain`. The convention is also to use lowercase for entry point names. So with these two things in mind, what we would suggest is to change your `setup.json` to the following:
```
    "entry_points": {
        "aiida.workflows": [
            "qecpworkchain.cp = aiida_QECpWorkChain.workflow:CpWorkChain"
        ],
        "aiida.calculations": [
            "qecpworkchain.deep_md_train = aiida_QECpWorkChain.deepmd:DeepMdTrainCalculation"
        ],
        "aiida.parsers": [
            "qecpworkchain.deep_md_train = aiida_QECpWorkChain.deepmd:DeepMdTrainParser"
        ]
    },
```
I chose `qecpworkchain` here as the entry point name prefix, so if you agree with that change, you should update your PR to change `entry_point_prefix` to `qecpworkchain`.

Reply:
Yes, it is ok! The idea is to provide just a workchain. The parser and the plugin of deepmd will became a different package (I forgot to remove it before uploading everything on github). I am planning to do it later!
It is a mess, sorry.
P.S. the aiida-quantumespresso parser for CP does not work with the last version of QE, see https://github.com/aiidateam/aiida-quantumespresso/pull/455